public/chirpy.js

```diff
@@ -490,15 +489,23 @@ function send(question, input) {
     var response = [];
     var userString = input ? input : d3.select('#userInput').property('value');
     var nonWhiteSpace = userString.search( /\S/ );
+
     if (!~nonWhiteSpace) {
         addUserChatLine('Skip');
-        response.push({
-            subject: question.type === 'Second Form Object' ? question.subject : null,
-            relationship: question.relationship,
-            object: question.type === 'Second Form Object' ? null : question.object,
+        const data = {
             cf: 100,
-            unanswered: true
-        });
+            unanswered: true,
+            relationship: question.relationship,
+        }
+
+        if (question.type === 'Second Form Subject') {
```

Comment:
Sorry I didn't spot this before but it looks like we've dropped skipping second form object questions? If it's "second form object" we'd send back a response without a subject and I think we reject it in that instance.

Reply:
I see, wasn't called in my example, it's back now.